src/index.html

```diff
@@ -2,22 +2,90 @@
 <html>
 <head>
     <title>ORCs Project</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+    <link rel="stylesheet" href="Stil.css">
```

Comment:
Please rename the Stil.css to some english word, like basic as this file is the basic sytlesheet.